kafka/conn.py

```diff
@@ -70,6 +71,9 @@ class BrokerConnection(object):
         'ssl_password': None,
         'api_version': (0, 8, 2),  # default to most restrictive
         'state_change_callback': lambda conn: True,
+        'sasl_mechanism': None,
+        'sasl_username': None,
+        'sasl_password': None
```

Comment:
> Is that the standard method for java?

As in "standard authentication method"? No. It's either PLAIN or GSSAPI (kerboros) which as far as I know does not use the terms "username" and "password". The (client) [config file](http://kafka.apache.org/documentation.html#security_sasl_plain_clientconfig) use the terms "username" and "password" with no prefix. 

But, now that you mention it, I do it think `sasl_plain_` is clearer.